packages/cosmic-swingset/lib/ag-solo/vats/lib-board.js

```diff
@@ -37,3 +37,5 @@ function makeBoard(seed = 0) {
 }
 
 export { makeBoard };
+
+/** @typedef {ReturnType<typeof makeBoard>} Board */
```

Comment:
I have a more comprehensive type for Board. Should that go here?
```
 * @typedef {Object} Board
 * @property {(id: string) => any} getValue
 * @property {(value: any) => string} getId
 * @property {(value: any) => boolean} has
 * @property {() => [string]} ids```

Reply:
That sounds fine to me, although does cosmic-swingset have exported types?

Reply:
Done.  We can add type exports to cosmic-swingset later.